File: bazinga.js
for (let num = 1; num < 100; num++) {
	output = ""

	if (num % 2 == 0) {
		output += "Baz"
	}

	if (num % 5 == 0) {
		output += "inga!"
	}

	console.log(output || num)
}

File: Bazinga.java
public class Bazinga{
  public static void main(String[] args){
    for(int i = 1; i < 100; i++){
      String output = "";

      if (i % 2 == 0){
        output += "Baz";
      }

      if (i % 5 == 0){
        output += "inga!";
      }

      System.out.println(output == "" ? i : output);
    }
  }
}
